scripts/tuple_tree_generator/tuple-tree-generate-typescript.py

```diff
@@ -33,6 +39,18 @@ def main(args):
         print(source)
 
 
+def prettify_typescript(source: str) -> str:
+    tmp_dir = Path(mkdtemp())
+    source_file = tmp_dir / "source.ts"
+    source_file.write_text(source)
+    run_opts = {"cwd": tmp_dir, "check": True, "stdout": DEVNULL, "stderr": DEVNULL}
+    run(["npm", "install", "prettier"], **run_opts)
+    run(["./node_modules/.bin/prettier", "--write", "source.ts"], **run_opts)
```

Comment:
Uhm, we need to install `npm` to do this. We need to think how we want to this.
Squash this in the previous commit, no need to fixup new files.

Reply:
Done